java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java

```diff
@@ -964,11 +972,16 @@ void testFromScalar() {
   void testFromScalarNull() {
     final int rowCount = 4;
     for (DType.DTypeEnum type : DType.DTypeEnum.values()) {
-      if (type == DType.DTypeEnum.EMPTY || type == DType.DTypeEnum.LIST || type == DType.DTypeEnum.STRUCT
-          || type == DType.DTypeEnum.DECIMAL32 || type == DType.DTypeEnum.DECIMAL64) {
+      if (type == DType.DTypeEnum.EMPTY || type == DType.DTypeEnum.LIST || type == DType.DTypeEnum.STRUCT) {
         continue;
       }
-      try (Scalar s = Scalar.fromNull(DType.create(type));
+      DType dType;
+      if (type == DType.DTypeEnum.DECIMAL32 || type == DType.DTypeEnum.DECIMAL64) {
+        dType = DType.create(type, -8);
```

Comment:
Why -8?

Reply:
It's just a random number to invoke factory method for decimal types.

Reply:
nit: The could we have a small comment to say that then?  A magic number with no explanation makes it hard to tell if that is important in some way or not.

Reply:
Comment added.